Algoritmos/com.busca.linear/src/Main.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        var sc = new Scanner(System.in);
+
+        int[] vetor = new int[5];
+        vetor[0] = 5;
+        vetor[1] = 1;
+        vetor[2] = 2;
+        vetor[3] = 3;
+        vetor[4] = 4;
+
+        boolean isNumero = false;
+        int numeroLocalizado = 0;
+
+        System.out.println("insira o numero a ser buscado: ");
+        int numeroBuscado = sc.nextInt(); // criada uma variável pra armazenar o numero que será buscado pelo algoritmo
+
+        for (int i : vetor) {
```

Comment:
Para aumentar a legibilidade do for sugiro:
```suggestion
                for (int numero : vetor) {
            if (numero == numeroBuscado) {
                isNumeroEncontrado = true;
                break;
            }
        }
```